tests/checkup_test.go

```diff
@@ -159,21 +158,21 @@ func newKiagnoseConfigMapAccessRole() *rbacv1.Role {
 	}
 }
 
-func newKiagnoseConfigMapAccessRoleBinding() *rbacv1.RoleBinding {
+func newKiagnoseConfigMapAccessRoleBinding(roleName, serviceAccountName string) *rbacv1.RoleBinding {
 	return &rbacv1.RoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: testKiagnoseConfigMapAccessRoleBindingName,
+			Name: roleName,
```

Comment:
shouldn't we use for generality's sake give it a different variable `roleBindingName`? it would make this function for general

Reply:
Good idea.
Done.